source/WorkingPractices/metadata_guidance.rst

```diff
@@ -12,20 +12,58 @@ with a consistent set of information, such as follows:
 
 .. code-block::
 
-    [namelist:<namelist>=<variable>]
-    description=<what is the variable for?>
-    help=<detailed information about the variable>
-    sort-key=<XY>
-    etc
+   [namelist:<namelist>=<variable>]
+   description=<what is the variable for?>
+   help=<detailed information about the variable>
+   sort-key=<XY>
+   url=<http://jules-lsm.github.io/latest/namelists/<namelist>.nml.html#<NAMELIST>::<item>>
+   etc
 
-then each input is presented as:
+where JULES namelist items should only contain the ``url=`` field and
+not the ``help=`` field regardless of the repository they reside
+in. Each input is presented in the GUI as:
 
 .. code-block::
 
     variable name
     short description of the variable name
 
-Help text is displayed when one clicks the variable name.
+Either the help text is displayed or web help opened when one clicks
+the variable name. If both fields exist the Web help is opened
+preferentially. If a ``url=`` is present the ``help=`` field should be
+removed to avoid duplication.
+
+.. _shared-namelists:
+
+Shared namelists
+----------------
+
+Shared namelist items, or items which exist in more than one
+repository, should have identical metadata regardless of the
+repository where they reside (e.g. JULES items in the UM
+metadata). The only caveats are where a ``trigger`` or a ``fail-if``
+reference a namelist item from the parent model (e.g. UM).
+
+Shared JULES metadata is in the process of being migrated to
+`rose-meta/jules-shared
+<https://code.metoffice.gov.uk/trac/jules/browser/main/trunk/rose-meta/jules-shared>`_,
+which resides in the JULES repository. The sub-directories are
+imported by **rose-meta/um-atmos** and **rose-meta/jules-standalone**
+and is manually synced with a copy in LFRic. Please see the `Sharing
+JULES metadata
```

Comment:
I think this should either be "Please see `Sharing JULES metadata`" or "Please see the `Sharing JULES metadata` wiki"

Reply:
Changed.